pkg/chrootuser/user.go

```diff
@@ -18,15 +18,19 @@ var (
 // it will use the /etc/passwd and /etc/group files inside of the rootdir
 // to return this information.
 // userspec format [user | user:group | uid | uid:gid | user:gid | uid:group ]
-func GetUser(rootdir, userspec string) (uint32, uint32, error) {
+func GetUser(rootdir, userspec string) (uint32, uint32, string, error) {
 	var gid64 uint64
 	var gerr error = user.UnknownGroupError("error looking up group")
-
+	usr, uerr := user.Current()
```

Comment:
Nit: why uerr versus err?

Reply:
just to specify user error and we're using the same name below at line 35 in pre-existing code.